examples/test-cra/src/storyshots.test.js

```diff
@@ -0,0 +1,5 @@
+import initStoryshots, { renderOnly } from 'storyshots';
+
+initStoryshots();
+
+// initStoryshots({ test: renderOnly });
```

Comment:
It's basically a way of testing `renderOnly` (until we figure out a better testing strategy). I'll try to make it better.